Jenkinsfile

```diff
@@ -655,7 +511,9 @@ pipeline {
                         }
                     }
                     steps {
-                        sconsBuild clean: "_build.external${arch}", COMPILER: "clang",
+                        sconsBuild clean: "_build.external${arch}",
+                                   prebuild: 'rm -rf bandit.xml',
```

Comment:
Should this just be moved to` sconsBuild()` as part of the step's general pre-clean?  It seems to be repeated in here quite a bit.

Reply:
I plan to look at doing those optimizations in a later PR.